src/formatDate.js

```diff
@@ -50,7 +50,41 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[3]);
```

Comment:
Will be better if you put fromFormat[3] in the variable. 
For example: const separator = fromFormat[3].